projects/fsastorefrontlib/src/cms-components/dynamic-forms/form-components/dynamic-select/dynamic-select.component.ts

```diff
@@ -33,10 +37,27 @@ export class DynamicSelectComponent extends AbstractFormComponent
   ngOnInit() {
     super.ngOnInit();
     if (this.config.apiValue) {
+      this.configureApiValueForCategory();
       this.setFormControlValuesFromAPI();
     }
   }
 
+  configureApiValueForCategory() {
+    if (!this.config.apiValue.url.includes(CATEGORY_CODE + '=')) {
+      this.subscription.add(
+        this.categoryService
+          .getActiveCategory()
+          .pipe(
+            map(activeCategory => {
+              this.config.apiValue.url +=
+                '?' + CATEGORY_CODE + '=' + activeCategory;
```

Comment:
It looks cleaner with string interpolation but you can leave it as it is

```suggestion
              this.config.apiValue.url += `?${CATEGORY_CODE}=${activeCategory}`
```
or
```suggestion
              this.config.apiValue.url = `${this.config.apiValue.url}?${CATEGORY_CODE}=${activeCategory}`
```